ssh/DOCS.md

```diff
@@ -31,7 +31,7 @@ You can access the web terminal by clicking the "Open Web UI" button on this add
 
 ### SSH Server Connection
 
-To connect using an SSH client, such as PuTTY, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
+Remote SSH access from the network is disabled by default (See Network below).  To connect using an SSH client, such as PuTTY or Linux terminal, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
```

Comment:
Minor tweaks:

```suggestion
Remote SSH access from the network is disabled by default (see Network below). To connect using an SSH client, such as PuTTY or Linux terminal, you need to supply additional configuration for this add-on. To enable SSH connectivity, you need to:
```

Additional concern: 

> To connect using an SSH client, such as PuTTY or Linux terminal

The Linux terminal itself is not an SSH client. The modified section below contains the same error.

Reply:
Good point, perhaps substitute "PuTTY or ssh from a shell command" ...  could even drop that change if it's not adding value.